_sass/module/_office-election.scss

```diff
@@ -1,5 +1,10 @@
 .office-election__header {
   border-bottom: 2px $color-gold solid;
+
+  h2 {
+    font-weight: $font-weight-normal;
+  }
```

Comment:
This rule seems a little funny to me. Why is the header on the office-election different than any other page?